### PR TITLE
[multimodal] wrap the dataset_mmlab/__init__.py with try except

### PR DESCRIPTION
**Pull request by FANGAreNotGnu**
Address the comments in https://github.com/autogluon/autogluon/pull/3266

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: multimodal/src/autogluon/multimodal/data/dataset_mmlab/__init__.py [hunk 1]
```diff
@@ -1 +1,4 @@
-from .multi_image_mix_dataset import MultiImageMixDataset
+try:
+    from .multi_image_mix_dataset import MultiImageMixDataset
+except ImportError:
```
[the remark this answers is not in the document]
Comment (zhiqiangdon) [reply multimodal/src/autogluon/multimodal/data/dataset_mmlab/__init__.py:3]: I guess a safer way is just removing `ImportError`. Remember double-checking the installation for the nightly build.

[the remark this answers is not in the document]
Comment (zhiqiangdon) [reply multimodal/src/autogluon/multimodal/data/dataset_mmlab/__init__.py:3]: Right. Also need to check whether the nightly build works without installing the mmlab packages.

**Review by zhiqiangdon — APPROVED** [review]
LGTM. Remember double-check whether the nightly build works without installing mmlab packages.

File: multimodal/src/autogluon/multimodal/data/dataset_mmlab/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (FANGAreNotGnu) [reply multimodal/src/autogluon/multimodal/data/dataset_mmlab/__init__.py:3]: Sure. I'll check with my repo again and will merge this if it works correctly.

[the remark this answers is not in the document]
Comment (FANGAreNotGnu) [reply multimodal/src/autogluon/multimodal/data/dataset_mmlab/__init__.py:3]: Checked for this branch in a new virtual environment (Linux). For object detection it shows installation guidance correctly, and for other tasks it works correctly as well.

**Pull request merged**